arrows/ocv/feature_detect_extract_SURF.cxx

```diff
@@ -195,7 +195,8 @@ ::set_configuration(vital::config_block_sptr config)
 #ifndef KWIVER_HAS_OPENCV_VER_3
   p_->update( detector );
 #else
-  p_->update( detector.dynamicCast<cv_SURF_t>() );
+  // version 3.x doesn't have parameter update methods
```

Comment:
I think this is a perfectly reasonable fix and we should go with it if it works.  However this comment is not accurate.  While SIFT does not have any update method, SURF does.  We are just deciding not to use them in this case.  We may want to do like we do for SIFT and just make the whole `update` function ifdef'ed out for OpenCV 3.

Reply:
Done